--- TransportWebAuthn.js	1985-10-26 03:15:00.000000000 -0500
+++ TransportWebAuthnMod.js	2019-05-02 10:21:29.013725600 -0400
@@ -12,6 +12,8 @@
 
 var _errors = require("@ledgerhq/errors");
 
+var regeneratorRuntime = require("@babel/runtime/regenerator");
+
 var _scrambling = require("@ledgerhq/devices/lib/scrambling");
 
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
@@ -179,4 +181,4 @@
 };
 
 exports.default = TransportWebAuthn;
-//# sourceMappingURL=TransportWebAuthn.js.map
\ No newline at end of file
+//# sourceMappingURL=TransportWebAuthn.js.map
